messages/store.json

```diff
@@ -134,14 +134,16 @@
         "map": {
             "cmdDescription": "Maps an extension to a specific webstore",
             "StoreNameFlagDescription": "The name of the web store to map the extension",
-            "errStoreName": "Web-storename is not valid",
-            "errStoreId": "Store Id is not valid",
-            "nameAlreadyExists": "Registered extension name already exists: ",
+            "errStoreName": "Cannot find wesbtore with the name: ",
```

Comment:
Have you run these messages by the commerce writer? I don't know enough about your product to  provide real feedback.  I'll do my best and point some stuff out, but it would be way better for a writer who understands this feature to review this content.

Question: is it "webstore" or "web store"?  I see both in the existing messages.  Please pick one and stick with it in all messages.

Edit:  Use "Can't" instead of "Cannot".